src/Core/tests/DeviceTests/Handlers/Layout/LayoutHandlerTests.iOS.cs

```diff
@@ -1,14 +1,44 @@
+using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Maui.DeviceTests.Stubs;
+using Microsoft.Maui.Graphics;
 using Microsoft.Maui.Handlers;
 using UIKit;
 
 namespace Microsoft.Maui.DeviceTests.Handlers.Layout
 {
 	public partial class LayoutHandlerTests
 	{
+		[Fact(DisplayName = "Shadow Initializes Correctly")]
```

Comment:
Could we include the same device test in Android?

Reply:
yes done